gateway/gateway.go

```diff
@@ -1146,50 +1144,51 @@ func (irh *ImportRequestHandler) ProcessRequest(gateway *HandleT, w *http.Respon
 		gateway.addToWebRequestQ(w, r, done, "batch", usersPayload[key], writeKey)
```

Comment:
Not related to the scope of the PR but I guess we should also be overriding the AnonymousID in the header so that we can route each user's payload to the appropriate worker